docker-compose.yml

```diff
@@ -412,6 +412,38 @@ services:
     volumes:
       - oracle:/opt/oracle/data
 
+  flink-jobmanager:
+    build: ./docker/flink
+    image: ibis-flink
+    environment:
+      - |
+        FLINK_PROPERTIES=
+        jobmanager.rpc.address: flink-jobmanager
+    ports:
+      - 8081:8081
+    command: jobmanager
+    networks:
+      - flink
+
+  flink:
+    build: ./docker/flink
+    image: ibis-flink
+    environment:
+      - |
+        FLINK_PROPERTIES=
+        jobmanager.rpc.address: flink-jobmanager
+        taskmanager.numberOfTaskSlots: 2
+        taskmanager.memory.process.size: 2048m
+        taskmanager.memory.network.fraction: 0.4
+        taskmanager.memory.network.min: 512mb
+        taskmanager.memory.network.max: 2gb
+    depends_on:
+      - flink-jobmanager
+    command: taskmanager
+    scale: 1
```

Comment:
I'm not sure if I understand your question. This `flink` entry is used to start a flink taskmanager container, which depends on jobmanager container. And this container is the one that runs all flink unit tests. Which part do you think is unnecessary?

Reply:
Sorry! I mean the `scale: 1` bit 😄

Reply:
oh I thought this specific property means only 1 taskmanager is spun up, which should be necessary?

Reply:
I looked into it out of curiosity, found a couple things:

1. It seems `scale` is deprecated in favor of `replicas` (under `deploy` key).
2. `replicas` only takes effect if deployed in swarm mode.

So maybe it wouldn't make a difference? I was looking for the default `replicas`, but couldn't find it for now.

Reply:
Thanks for the due diligence! I'll remove this property in that case.

Reply:
You shouldn't need to deal with scaling or replicas at all, since this compose file is not going to be used for docker swarm or any kind of production deployment.

Plus, we're not using it anywhere else in the file :)